pysages/backends/openmm.py

```diff
@@ -169,9 +180,9 @@ def bind(context, sampling_method, force = Force(), **kwargs):
     #
     force.add_to(context)
     wrapped_context = ContextWrapper(context, force)
-    indices, momenta, bias = build_helpers(wrapped_context.view)
+    indices, momenta, bias, restore = build_helpers(wrapped_context.view)
```

Comment:
I see, that we are returning here tuples a couple of times.
Maybe it is more flexible to return here a dictionary or named_tuple instead from the helpers.

If we have to adjust to include more returns, we just add another field to the dict or namedtuple, without having to refactor old code, because it returns more values now.